src/components/blog/ImageBlock/ImageBlock.module.css

```diff
@@ -0,0 +1,17 @@
+.image {
+  display: block;
```

Comment:
Setting the margin to `auto` wouldn't center the image unless I set `display` to `block`, but since `block` is concerning here I just replaced it with `flex` and `align-items`

Reply:
Ah, I see. Yeah, I think having the parent be a flex container makes sense. I'm in general a bit wary of directly messing with `img` elements, so this looks like a good alternative to me.